src/pages/settings/Profile/Contacts/NewContactMethodPage.tsx

```diff
@@ -47,7 +48,8 @@ function NewContactMethodPage({loginList, route}: NewContactMethodPageProps) {
             const validateIfnumber = LoginUtils.validateNumber(phoneLogin);
             const submitDetail = (validateIfnumber || values.phoneOrEmail).trim().toLowerCase();
 
-            User.addNewContactMethodAndNavigate(submitDetail, route.params?.backTo);
+            User.saveNewContactMethodAndValidateAction(submitDetail);
```

Comment:
I think this is clearer as we do not validated it yet 
```suggestion
            User.saveNewContactMethodAndRequestValidationCode(submitDetail);
```